helix-term/src/commands.rs

```diff
@@ -1031,9 +1031,9 @@ fn goto_file_impl(cx: &mut Context, action: Action) {
         .collect();
     let primary = selections.primary();
     if selections.len() == 1 && primary.to() - primary.from() == 1 {
-        let current_word = movement::move_next_long_word_start(
+        let current_word = movement::move_prev_long_word_start(
             text.slice(..),
-            movement::move_prev_long_word_start(text.slice(..), primary, 1),
+            movement::move_next_long_word_start(text.slice(..), primary, 1),
```

Comment:
With this change I see a regression:

```
helix-term/src/main.rs
import "helix-term/src/main.rs"
```

With the cursor on the the last `s` on the first line, `gf` opens "import" and not the path

Reply:
> With this change I see a regression:
> 
> ```
> helix-term/src/main.rs
> import "helix-term/src/main.rs"
> ```
> 
> With the cursor on the the last `s` on the first line, `gf` opens "import" and not the path

I changed approach and it should be working correctly on all cases now.